src/couchConfig.js

```diff
@@ -0,0 +1,361 @@
+/*
+Copyright 2017 OCAD University
+Licensed under the Educational Community License (ECL), Version 2.0 or the New
+BSD license. You may not use this file except in compliance with one these
+Licenses.
+You may obtain a copy of the ECL 2.0 License and BSD License at
+https://raw.githubusercontent.com/fluid-project/fluid-couch-config/master/LICENSE.txt
+*/
+
+"use strict";
+
+var fluid = require("infusion");
+var isEqual = require("underscore").isEqual;
+var size = require("underscore").size;
+
+var sjrk = fluid.registerNamespace("sjrk");
```

Comment:
Please update the namespace to `"fluid.couchConfig"`, replacing `"srjk.server.couchConfig"`.